src/client/js/components/More/More.jsx

```diff
@@ -3,16 +3,19 @@ import { Link } from 'react-router-dom'
 import { stylesheet, classes } from 'typestyle'
 import { colors } from '../../styles/common'
 
-const buildLinks = pathname => {
+const PAGE_SIZE = 30 // TODO: this should be shared config and either passed in request or shared with server
```

Comment:
Is there a way we can fix that TODO?

Reply:
this one would require a refactor of the server/api which I think warrants a separate pr